index.bs

```diff
@@ -9024,6 +9070,386 @@ worker=] algorithm:
 
 </div>
 
+## The storage Module ## {#module-storage}
+
+The <dfn export for=modules>storage</dfn> module contains functionality and
+events related to storage.
+
+A <dfn>storage partition</dfn> is a namespace within which the user agent may
+organize persistent data such as [=cookies=] and local storage.
+
+A <dfn>storage partition key</dfn> uniquely identifies a [=storage partition=].
+
+### Definition ### {#module-storage-definition}
+
+[=Remote end definition=]
+
+<pre class="cddl remote-cddl">
+StorageCommand = (
+  storage.GetCookies //
+  storage.SetCookie //
+  storage.DeleteCookies
+)
+</pre>
+
+[=Local end definition=]
+
+<pre class="cddl local-cddl">
+StorageResult = (
+  storage.GetCookiesResult /
+  storage.SetCookieResult /
+  storage.DeleteCookiesResult
+)
+</pre>
+
+### Types ### {#module-storage-types}
+
+#### The storage.PartitionKey Type #### {#type-storage-PartitionKey}
+
+[=Local end definition=]
+
+<pre class="cddl local-cddl remote-cddl">
+storage.PartitionKey = {
+    userContext: text,
```

Comment:
I believe this parameter is either optional or nullable, to point out the default user context.

Reply:
Yes, everything here should be optional. If the implementation doesn't support the key it shouldn't return it.

Reply:
Addressed via commit 35d7fad.